src/uct/ib/rc/accel/rc_mlx5.inl

```diff
@@ -226,6 +230,14 @@ uct_rc_mlx5_iface_rx_mp_context_from_hash(uct_rc_mlx5_iface_common_t *iface,
 
     last = cqe->byte_cnt & htonl(UCT_IB_MLX5_MP_RQ_LAST_MSG_FLAG);
 
+    if (ucs_likely(ucs_test_all_flags(cqe->byte_cnt,
+                                      htonl(UCT_IB_MLX5_MP_RQ_LAST_MSG_FLAG) |
+                                      htonl(UCT_IB_MLX5_MP_RQ_FIRST_MSG_FLAG)))) {
+        ucs_assert(md->flags & UCT_IB_MLX5_MD_FLAG_MP_XRQ_FIRST_MSG);
```

Comment:
maybe we can remove this assert together with `ib_striding_wq_cq_first_indication` capability checking

Reply:
I vote to have the capability